cmd/collector/app/zipkin/http_handler_test.go

```diff
@@ -75,20 +87,13 @@ func TestViaClient(t *testing.T) {
 	server, handler := initializeTestServer(nil)
 	defer server.Close()
 
-	zipkinSender, err := zipkinTransport.NewHTTPTransport(
-		server.URL+`/api/v1/spans`,
-		zipkinTransport.HTTPBatchSize(1),
-	)
+	zexp, err := zipkin.New(server.URL + `/api/v1/spans`)
```

Comment:
Actually Yuri, I shut downed the exporter after sending the spans frequently. Thanks for quick kind help